### PR TITLE
Peter/roann edit branch

### PR DESCRIPTION
**Pull request by Petertaylor9999**

### CODE REVIEW

**Comment by bennage** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/building-blocks/extending-templates/collector.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/collector.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/collector?branch=pr-en-us-162) | [Details](#user-content-docs%2Fbuilding-blocks%2Fextending-templates%2Fcollector.md)
[docs/building-blocks/extending-templates/toc.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/toc.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/objects-as-parameters?branch=pr-en-us-162) | [Details](#user-content-docs%2Fbuilding-blocks%2Fextending-templates%2Ftoc.md)
[docs/building-blocks/_images/collector-transformer.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/collector-transformer.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/collector?branch=pr-en-us-162) |
[docs/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png?branch=pr-en-us-162) |
[docs/building-blocks/_images/firstVNet-subnets.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/firstVNet-subnets.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |
[docs/building-blocks/_images/nic1-ipconfigurations.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/nic1-ipconfigurations.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/index.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/index?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/objects-as-parameters.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/objects-as-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/objects-as-parameters?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/update-resource.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/update-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |

<a name="docs%2Fbuilding-blocks%2Fextending-templates%2Fcollector.md"></a>
### [docs/building-blocks/extending-templates/collector.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/collector.md)
 - **Line 27**: **[Warning]** Invalid file link:(~/building-blocks/extending-templates/serial-copy-loop.md).

<a name="docs%2Fbuilding-blocks%2Fextending-templates%2Ftoc.md"></a>
### [docs/building-blocks/extending-templates/toc.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/toc.md)
 - **[Warning]** Invalid file link:(~/building-blocks/extending-templates/conditional-deploy.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C7%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201706072232519908-162%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/building-blocks/_images/collector-transformer.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/collector-transformer.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/collector?branch=pr-en-us-162) |
[docs/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png?branch=pr-en-us-162) |
[docs/building-blocks/_images/firstVNet-subnets.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/firstVNet-subnets.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |
[docs/building-blocks/_images/nic1-ipconfigurations.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/nic1-ipconfigurations.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/collector.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/collector.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/collector?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/conditional-deploy.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/conditional-deploy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/conditional-deploy?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/index.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/index?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/objects-as-parameters.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/objects-as-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/objects-as-parameters?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/toc.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/toc.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/objects-as-parameters?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/update-resource.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/update-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C7%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201706072238415958-162%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/building-blocks/_images/collector-transformer.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/collector-transformer.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/collector?branch=pr-en-us-162) |
[docs/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png?branch=pr-en-us-162) |
[docs/building-blocks/_images/firstVNet-subnets.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/firstVNet-subnets.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |
[docs/building-blocks/_images/nic1-ipconfigurations.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/nic1-ipconfigurations.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/collector.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/collector.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/collector?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/conditional-deploy.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/conditional-deploy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/conditional-deploy?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/index.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/index?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/objects-as-parameters.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/objects-as-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/objects-as-parameters?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/toc.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/toc.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/collector?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/update-resource.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/update-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C8%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201706082336158297-162%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/building-blocks/_images/collector-transformer.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/collector-transformer.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/collector?branch=pr-en-us-162) |
[docs/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png?branch=pr-en-us-162) |
[docs/building-blocks/_images/firstVNet-subnets.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/firstVNet-subnets.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |
[docs/building-blocks/_images/nic1-ipconfigurations.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/nic1-ipconfigurations.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/collector.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/collector.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/collector?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/conditional-deploy.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/conditional-deploy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/conditional-deploy?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/index.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/index?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/objects-as-parameters.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/objects-as-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/objects-as-parameters?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/toc.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/toc.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/objects-as-parameters?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/update-resource.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/update-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C9%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201706091951336709-162%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Petertaylor9999** [comment]
@telmosampaio

File: docs/building-blocks/extending-templates/collector.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+> [!IMPORTANT]
+> This approach requires that you have a deep understanding of Resource Manager templates and functions.
+
+Let's take a look at how we can implement a property collector and transformer with an example that deploys a [network security group (NSG)][nsg]. The diagram below shows the relationship between our templates and our resources within those templates:
+
+![property collector and transformer architecture](../_images/collector-transformer.png)
+
+Our **calling template** includes two resources:
+* a template link that invokes our **collector template**.
+* an NSG.
```
Comment (telmosampaio) [inline docs/building-blocks/extending-templates/collector.md]: Is that an NSG resource? As in the exact resource shape expected by the RP? If so, what are we transforming? If not, can we be more specific about what we are doing here?

File: docs/building-blocks/extending-templates/collector.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+
+Our **collector template** includes two resources:
+* an **anchor** resource.
+* a template link that invokes the transform template in a copy loop.
+
+Our **transform template** includes a single resource: an empty template with a variable that transforms our `source` JSON to the JSON schema expected by our NSG resource in the **main template**.
+
+## Parameter object
+
+We'll be using our `securityRules` parameter object from [objects as parameters][objects-as-parameters]. Our **transform template** will transform each object in the `securityRules` array into the JSON schema expected by the NSG resource in our **calling template**.
```
Comment (telmosampaio) [inline docs/building-blocks/extending-templates/collector.md]: Are we ok using "our" all the time? Who are we?

File: docs/building-blocks/extending-templates/collector.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+* an NSG.
+
+Our **collector template** includes two resources:
+* an **anchor** resource.
+* a template link that invokes the transform template in a copy loop.
+
+Our **transform template** includes a single resource: an empty template with a variable that transforms our `source` JSON to the JSON schema expected by our NSG resource in the **main template**.
+
+## Parameter object
+
```
Comment (telmosampaio) [inline docs/building-blocks/extending-templates/collector.md]: BTW, this was the winning paragraph for me. I only understood what we were trying to do when I got here. I suggest maybe moving the first diagram and list of elements to after this example.

File: docs/building-blocks/extending-templates/conditional-deploy.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+author: petertay
+ms.date: 06/09/2017
+
+---
+
+# Conditionally deploy a resource in an Azure Resource Manager template
+
+There are some scenarios in which you need to design your template to deploy a resource based on a condition, such as whether or not a parameter value is present. For example, your template may deploy a virtual network and include parameters to specify other virtual networks for peering. If you've not specified any parameter values for peering, you don't want Resource Manager to deploy the peering resource.
+
+To accomplish this, use the [`condition` element][azure-resource-manager-condition] in the resource to test the length of your parameter array. If the length is zero, return `false` to prevent deployment, but for all values greater than zero return `true` to allow deployment.
```
Comment (telmosampaio) [inline docs/building-blocks/extending-templates/conditional-deploy.md]: Did you really mean to have [`condition` element]? It renders weird. Should you remove the `s

File: docs/building-blocks/extending-templates/conditional-deploy.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+Our `name` for the inline template is made unique by concatenating the current iteration of the `copyIndex()` with the prefix `vnp-`. 
+
+The `condition` element specifies that our resource should be processed when the `greater()` function evaluates to `true`. Here, we're testing if the `virtualNetworkPeerings` parameter array is `greater()` than zero. If it is, it evaluates to `true` and the `condition` is satisfied. Otherwise, it's `false`.
+
+Next, we specify our `copy` loop. It's a `serial` loop that means the loop is done in sequence, with each resource waiting until the last resource has been deployed. The `count` property specifies the number of times the loop iterates. Here, normally we'd set it to the length of the `virtualNetworkPeerings` array because it contains the parameter objects specifying the resource we want to deploy. However, if we do that, validation will fail if the array is empty because Resource Manager notices that we are attempting to access properties that do not exist. We can work around this, however. Let's take a look at the variables we'll need:
```
Comment (telmosampaio) [inline docs/building-blocks/extending-templates/conditional-deploy.md]: Should we explain why this is a serial loop?

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/building-blocks/_images/collector-transformer.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/collector-transformer.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/collector?branch=pr-en-us-162) |
[docs/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/_images/extending-resource-manager-templates-sequential-loop.png?branch=pr-en-us-162) |
[docs/building-blocks/_images/firstVNet-subnets.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/firstVNet-subnets.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |
[docs/building-blocks/_images/nic1-ipconfigurations.png](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/_images/nic1-ipconfigurations.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/collector.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/collector.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/collector?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/conditional-deploy.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/conditional-deploy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/conditional-deploy?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/index.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/index?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/objects-as-parameters.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/objects-as-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/objects-as-parameters?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/toc.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/toc.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/objects-as-parameters?branch=pr-en-us-162) |
[docs/building-blocks/extending-templates/update-resource.md](https://github.com/mspnp/architecture-center/blob/peter/roann-edit-branch/docs/building-blocks/extending-templates/update-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/building-blocks/extending-templates/update-resource?branch=pr-en-us-162) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C12%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201706121705175250-162%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**